<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesWidget</class>
 <widget class="QWidget" name="FilesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>587</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView {
	border: 0px;
	border-right: 1px solid rgba(0, 0, 0, 100);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="MyTableView" name="files">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(243, 246, 250);
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>22</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {
	border: 0px;
	border-right: 1px solid rgba(0, 0, 0, 100);
	background-color: #b0b0b0;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="MyLabel" name="sortByNew">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #e0e0e0;
border: 1px;
border-radius: 2px;
font-size: 11px;
margin-right: 2px;
padding-left: 3px;
padding-right: 3px;
padding-top: 2px;
padding-bottom: 2px;</string>
        </property>
        <property name="text">
         <string>New</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="sortByName">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #c0c0c0;
border: 1px;
border-radius: 2px;
font-size: 11px;
margin-right: 2px;
padding-left: 3px;
padding-right: 3px;
padding-top: 2px;
padding-bottom: 2px;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="sortByLastAdded">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #c0c0c0;
border: 1px;
border-radius: 2px;
font-size: 11px;
margin-right: 2px;
padding-left: 3px;
padding-right: 3px;
padding-top: 2px;
padding-bottom: 2px;</string>
        </property>
        <property name="text">
         <string>Date Added</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="sortBySeen">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #c0c0c0;
border: 1px;
border-radius: 2px;
font-size: 11px;
margin-right: 2px;
padding-left: 3px;
padding-right: 3px;
padding-top: 2px;
padding-bottom: 2px;</string>
        </property>
        <property name="text">
         <string>Seen</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="sortByYear">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #c0c0c0;
border: 1px;
border-radius: 2px;
font-size: 11px;
margin-right: 2px;
padding-left: 3px;
padding-right: 3px;
padding-top: 2px;
padding-bottom: 2px;</string>
        </property>
        <property name="text">
         <string>Year</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>smallWidgets/MyLabel.h</header>
  </customwidget>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>smallWidgets/MyTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <signal>movieSelected(Movie*)</signal>
  <signal>noMovieSelected()</signal>
 </slots>
</ui>
